style.css

```diff
@@ -412,3 +424,199 @@ button {
   padding: 12px;
   color: #396df2;
 }
+
+.navDesk {
+  display: none;
+}
+
+@media screen and (max-width: 480px) {
+  .card5,
+  .card6,
+  .card7 {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 992px) {
+  .menu-icon {
+    display: none;
+  }
+
+  .first-page {
+    background-image: none;
+  }
+
+  .logo {
+    width: 600px;
+    justify-content: start;
+    margin-top: 150px;
+    margin-left: 140px;
+  }
+
+  .header-container {
+    display: inline;
+  }
+
+  .navDesk {
+    display: flex;
+    width: 50%;
+    margin-left: 50%;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .menuDesk {
+    display: inline;
+  }
+
+  .menuDesk a {
+    font-family: Poppins, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    list-style: none;
+    margin-right: 16px;
+    text-decoration: none;
+  }
+
+  .headline {
+    background-image: url('src/header-shapes-mobile.svg');
+    background-size: 90%;
+    background-position: 0% 15%;
+    align-items: flex-start;
+    justify-content: center;
+    padding-left: 140px;
+    padding-bottom: 150px;
+    border-bottom-left-radius: 165px;
+  }
```

Comment:
- [ ] Kindly make the background take the whole width, to be like the Figma design. And here you are using the mob image you can use the desktop version image.

- [ ] Kindly make your header line `I'm Shaheda Glad to see you!` in two lines as in the Figma design.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/155703877-2383e823-d7a7-4abd-bc7d-f80fb5b26df6.png) | ![image](https://user-images.githubusercontent.com/35535130/155703913-1034cf63-9975-4e6b-8391-cd5cb67d96dc.png) |